### PR TITLE
CU-1yzke36 feat(shadows): add shadows vars

### PR DESCRIPTION
**Pull request by YossiSaadi**
adding vars to shadows instead of classes as instructed by @talkor 
I asked Shuju for if she wants to have variations for light/dark and if so we'll change accordingly

### CODE REVIEW

File: public/styles/colors.css [hunk 1]
```diff
@@ -88,20 +88,13 @@
...
-    box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.2);
-}
-
-.box-shadow-medium {
-    box-shadow: 0px 1px 8px rgba(0, 0, 0, 0.2);
-}
-
-.box-shadow-big {
-    box-shadow: 0px 12px 18px rgba(0, 0, 0, 0.12), 0px 5px 22px rgba(0, 0, 0, 0.16), 0px 7px 8px rgba(0, 0, 0, 0.2);
-}
```
Comment (yhamry) [inline public/styles/colors.css:107]: Why would you remove it?

Comment (yhamry) [reply public/styles/colors.css:107]: `box-shadow-big` - is being used for example

File: public/styles/colors.css [hunk 2]
```diff
@@ -88,20 +88,13 @@
         var(--gradient-middle) 0.01%,
         var(--gradient-to) 100%
     );
+
+    /*** Shadows ***/
+    --box-shadow-small: 0 1px 3px rgba(0, 0, 0, 0.2);
+    --box-shadow-medium: 0 1px 8px rgba(0, 0, 0, 0.2);
+    --box-shadow-big: 0 12px 18px rgba(0, 0, 0, 0.12), 0 5px 22px rgba(0, 0, 0, 0.16), 0 7px 8px rgba(0, 0, 0, 0.2);
```
Comment (yhamry) [inline public/styles/colors.css:95]: You can add these and use them in the classes as well if you'd like

File: public/styles/colors.css (hunk 1 above)
Comment (YossiSaadi) [reply public/styles/colors.css:107]: @yhamry I'm switching the classes to vars also on the `swimm` repo

**Review by yhamry — APPROVED** [review]

**Pull request merged**